src/components/Workspace.jsx

```diff
@@ -64,14 +68,28 @@ function mapStateToProps(state) {
     currentProject: getCurrentProject(state),
     errors: state.get('errors').toJS(),
     runtimeErrors: state.get('runtimeErrors').toJS(),
+    isMaskingOutput: state.getIn(['ui', 'workspace', 'isMaskingOutput']),
```

Comment:
`isMaskingOutput` at first blush made me think that the output is visually occluded—maybe something like `isCapturingOutputPointerEvents`?

Or, alternatively, `isDraggingColumnDivider`? (I.e., let the individual components decide what to do about that fact)